web/src/components/eventslist-component/eventspanel.js

```diff
@@ -0,0 +1,120 @@
+import React, {Component} from 'react';
+import Cards from './cards';
+import {
+  ToggleButton,
+  ToggleButtonGroup,
+  Jumbotron
+} from 'react-bootstrap';
+
+
+
+import {gql} from 'apollo-boost';
+import { Query } from "react-apollo";
+
+/* Query requests event list to server */
+const EVENT_LIST_REQUEST_QUERY = gql`
+    query{
+        events(pageSize: 9){
+            cursor,
+            hasMore,
+            events{
+                id,
+                title,
+                description
+            }
+        }
+    }`;
+
+class Eventspanel extends Component {
+  constructor(props) {
+    // get area selection from props
+    super(props);
+
+    // save the arrays of filter names and filter toggle(selected or not) in state
+    const filterNames = ['BookClub', 'Movie', 'Wine', 'SciFi', 'Sport'];
```

Comment:
In another PR, these are stored as an integer to indicate the enum. Shouldn't we have a mapping somewhere between these strings and the integers?

Reply:
I understand! We'll mapping it later because our db is not constructed yet :)